src/pages/embed.vue

```diff
@@ -190,6 +192,15 @@ const route = useRoute()
 const timePeriod = ref(simulationArea.timePeriod)
 const clockEnabled = ref(simulationArea.clockEnabled)
 
+// Embed user preferences
+const theme = ref(route.query.theme)
+const displayTitle = ref(route.query.display_title ? route.query.display_title === 'true' : false);
+const clockTime = ref(route.query.clock_time ? route.query.clock_time === 'true' : true);
+const fullscreen = ref(route.query.fullscreen ? route.query.fullscreen === 'true' : true);
+const zoomInOut = ref(route.query.zoom_in_out ? route.query.zoom_in_out === 'true' : true);
```

Comment:
Let's have some thumb rule for naming convention, for any boolean type, start with `is`/ `has`